src/common/gui/nav/ViewSlider.ts

```diff
@@ -73,7 +73,7 @@ export class ViewSlider implements Component<ViewSliderAttrs> {
 	}
 	private getSideColDom: () => HTMLElement | null = () => this.viewColumns[0].domColumn
 
-	constructor(private readonly viewColumns: ViewColumn[]) {
+	constructor(private readonly viewColumns: ViewColumn[], private readonly enableDrawer: boolean = true) {
```

Comment:
I don't understand why we need it, can't we just not add the first (foreground) column?

Reply:
I tried that as well, but an empty drawer was still there. I then managed to not have the drawer anymore, but the gestures were still there and allowed the screen to get shifty, so I had to disable those as well.